app/app/context.py

```diff
@@ -14,5 +14,7 @@ def insert_settings(request):
         'trackjs_token': settings.TRACKJS_TOKEN,
```

Comment:
we should lose trackJS ... that is the old tool used to track production JS errors.